### ISSUE TITLE
Is there any documentation?

### ISSUE THREAD

**Issue by JunwenBai** [seanbell/opensurfaces-segmentation-ui#2]
I am quite confused by some parts of the code. If there is any doc, could you share it with us?

**Comment by seanbell** [comment]
I don't believe I wrote individual docs for just the segmentation UI, but I'm happy to answer any questions you may have if you point me to specific issues you are having.

There is documentation for the larger OpenSurfaces project here: http://opensurfaces.cs.cornell.edu/docs/ (but it won't answer questions about this particular repo)
